### PR TITLE
new orthographies, CL switch, some reorg

### PR DESCRIPTION
**Pull request by ChristTrekker**
Broken previous (cancelled) pull request into more atomic units. Now you can look at the orthography changes separately from functionality changes. Esperanto change saved for later time.

### CODE REVIEW

File: src/FontFace.cpp [hunk 1]
```diff
@@ -340,8 +342,9 @@ bool FontFace::_checkOrthography( const OrthographyData *pData ){
 	//
 	// Spot check using "key" value first
 	// to avoid wasting time:
+	// No: Computers so fast now, why risk false positive?
 	//
-	if(!hasUnicodeValue(pData->key)) return false;
+	//if(!hasUnicodeValue(pData->key)) return false;
```
Comment (christopheradams) [inline src/FontFace.cpp]: If this is really no longer necessary, the code should be removed rather than commented out.

**Comment by christopheradams** [comment]
@piratical can you check this?

**Comment by christopheradams** [comment]
Can we increment the version number while we're at it? It's been at Version 1.01 for as long as I can remember.

**Comment by piratical** [comment]
OK, I removed the commented-out "spot check" code and bumped the version number to 1.2.

**Pull request merged**